lib/visual_call_graph/graph_manager.rb

```diff
@@ -32,8 +32,10 @@ def get_node_name(event)
     end
   end
 
-  def output(*args)
-    @g.output(*args)
+  def output
+    format = @options[:format] || :png
+    path   = @options[:path] || "#{Dir.pwd}/call_graph.#{format.to_s}"
+    @g.output(format.to_sym => path)
```

Comment:
Can you add a blank line between the local variables & the method call? I think it makes the code easier to read :)